### PR TITLE
[ADD] support_branding

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: support_branding/static/src/js/support_branding.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* 
+
+   Copyright (C) 2012 Therp BV
```
Comment (StefanRijnhart) [inline support_branding/static/src/js/support_branding.js]: Can you update the year ranges in all the copyright headers?

File: support_branding/__openerp__.py [hunk 2]
```diff
@@ -18,34 +18,22 @@
...
-
-To configure this module, please enter the values for thetwo
-variables 'support_name' and 'support_link' in the file
-support_branding/static/src/js/chrome.js.
-
-This module is compatible with OpenERP 6.1.
-""",
-    "category": "Therp web addons",
-    "version": "1.0r4",
+    "summary": "Adds you branding to an Odoo instance",
```
Comment (StefanRijnhart) [inline support_branding/__openerp__.py]: s/you/your/

**Comment by StefanRijnhart** [comment]
Very nice! Apart from the nags, can you put the database / hostname in the subject of the email?

**Comment by hbrunn** [comment]
@StefanRijnhart done, I also put the username in the subject

**Comment by StefanRijnhart** [comment]
Thanks! :+1:

**Pull request merged**